src/settings.js

```diff
@@ -47,7 +47,9 @@ var INVOKE_RUN = 1; // Whether we will run the main() function. Disable if you e
 var NO_EXIT_RUNTIME = 0; // If set, the runtime is not quit when main() completes (allowing code to
                          // run afterwards, for example from the browser main event loop).
 var MEM_INIT_METHOD = 0; // How to represent the initial memory content.
-                         // 0: keep array literal representing the initial memory data
+                         // 0: embed a base64 string literal representing the initial memory data;
+                         //    if using this in combination with HTML output, your Content Security
+                         //    Policy may need to be modified (see SINGLE_FILE for more details)
                          // 1: create a *.mem file containing the binary data of the initial memory;
                          //    use the --memory-init-file command line switch to select this method
                          // 2: embed a string literal representing that initial memory data
```

Comment:
how about adding to the comment for mode 2 that it is deprecated in favor of 0, which now uses base64 and is pretty efficient, etc.?